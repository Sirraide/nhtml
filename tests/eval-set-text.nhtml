title
p

eval { $('title').text = 'foobarbaz' }
eval { $('p').text = $('title').text }