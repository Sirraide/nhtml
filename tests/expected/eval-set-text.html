<title>foobarbaz</title>
<p>foobarbaz</p>
